internal/database/postgres/migration/000004_add-index-to-flight-id-on-tickets-table.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX idx_flight_id ON tickets(flight_id);
```

Comment:
NIT: `.down` files generally should contain "rollback" of the action you performed, so in case of creating index, `down` should drop said index. It's fine with indexes and such, but should be considered carefully with new fields, changing types of field, etc. Because some of the changes might be irreversible _logically_ if not physically